src/test/java/shop/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,52 @@
+package shop.dao;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import shop.db.DataStorage;
+import shop.model.Fruit;
+
+public class FruitDaoImplTest {
+    private static FruitDaoImpl fruitDao;
+    private static Fruit test_fruit;
+
+    @BeforeClass
+    public static void beforeAll() {
+        fruitDao = new FruitDaoImpl();
+        test_fruit = new Fruit("apple", 1);
+    }
+
+    @Test
+    public void fruitDao_getAll_ok() {
+        Fruit banana = new Fruit("Banana", 10);
+        Fruit orange = new Fruit("Orange", 5);
+        fruitDao.add(banana);
+        fruitDao.add(orange);
+        Assert.assertTrue(fruitDao.getAll().contains(banana));
+        Assert.assertTrue(fruitDao.getAll().contains(orange));
```

Comment:
you should not test several methods in 1 place
test `add()` separately and `getAll()` separately

you should be able to directly access storage